.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
looks like tsdx gave us prettier config already in package.json so we probably don't need this file